integrationtests/agent/adoption_test.go

```diff
@@ -35,276 +35,61 @@ data:
 var _ = Describe("Adoption", Label("adopt"), func() {
 	var (
 		namespace string
```

Comment:
Where is this namespace set?

Reply:
In `BeforeEach` for each `It` a few lines below.

```
	BeforeEach(func() {
		namespace = createNamespace() # <-- here
		env = adoptEnv{namespace: namespace, env: &specEnv{namespace: namespace}}
	})
```

Reply:
So it is :facepalm: thanks!